README.md

```diff
@@ -445,7 +445,7 @@ client.post('ref-data/isin', isin: ['US0378331005'], token: 'secret_token') # [{
 You can configure client options globally or directly with a `IEX::Api::Client` instance.
 
 ```ruby
-IEX::Api::Client.configure do |config|
```

Comment:
I didn't dig too deeply but going back to 1.2.0 before I started screwin' around there was no `IEX::Api::Client.configure` method. I believe this was a mistake in the docs between `IEX::Api.configure` and `IEX::Api::Client.new(…)`